tests/Doctrine/Tests/ORM/Tools/Export/annotation/Doctrine.Tests.ORM.Tools.Export.User.php

```diff
@@ -5,6 +5,11 @@
 /**
  * @Entity
  * @HasLifecycleCallbacks
+ * @EntityListeners({
+ *     Doctrine\Tests\ORM\Tools\Export\UserListener::class,
```

Comment:
There is a weird issue with this, the resulting string is `"\Doctrine\Tests\ORM\Tools\Export\UserListener"`, although `::class` notation normally results in class names without root namespace:

```
php > echo \SimpleXMLElement::class;
SimpleXMLElement
```